src/GovUk.Education.ExploreEducationStatistics.Content.Api/Startup.cs

```diff
@@ -68,23 +69,22 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseHsts(hsts => hsts.MaxAge(365).IncludeSubdomains());
             }
 
-            // Enable middleware to serve generated Swagger as a JSON endpoint.
-            app.UseSwagger();
-
-            // Enable middleware to serve swagger-ui (HTML, JS, CSS, etc.), 
-            // specifying the Swagger JSON endpoint.
-            app.UseSwaggerUI(c =>
+            if(env.IsDevelopment() || Environment.GetEnvironmentVariable("enableSwagger") == "True")
```

Comment:
I'm never sure whether we should be doing `Environment.GetEnvironmentVariable` or `Configuration.GetValue<bool>` here where Configuration is the IConfiguration in the Startup constructor. Perhaps someone else can comment, but the latter returns a bool.